src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -937,18 +937,16 @@ def getWorkloadArguments():
                                          "attr": "priority"},
                      "VoGroup": {"default": "unknown", "attr": "owner_vogroup"},
                      "VoRole": {"default": "unknown", "attr": "owner_vorole"},
-                     "Campaign": {"optional": False},
-                     "AcquisitionEra": {"default": "FAKE", "validate": acqname,
-                                        "assign_optional": False},
+                     "Campaign": {"optional": True}, # default value will be AcquisitionEra
```

Comment:
This will default to `'default': None, 'null': False`, which means it cannot be None. Did you try a case whether there is no `Campaign` in the create dict?